algorithms/rainbow/test.py

```diff
@@ -12,8 +12,11 @@
 
 from algorithms.rainbow.env import Env
 
-# Globals
-# TODO: explain what is Ts and Qs
+""" Global variables used to track the evaluation results
+Ts               - list of evaluation steps at each evaluation period
```

Comment:
I still think T is bad name (mentioned in other comment) here due to transformation T in the distributional algorithm and generally just being a bad name.

Reply:
Changed it to steps